### ISSUE TITLE
Weather volume should be adjustable

### ISSUE THREAD

**Issue by Jakeybobble** [space-wizards/space-station-14#43872]
I have a form of tinnitus where listening to white noise for a prolonged amount of time can leave my tinnitus stronger for days or even weeks. Since weather is on master volume, there's no way for me to turn off weather sounds without muting the game.  
It makes sense to make weather use ambience volume, or to introduce a weather volume slider.
